vm/src/obj/objfloat.rs

```diff
@@ -258,6 +269,17 @@ impl PyFloat {
         )
     }
 
+    #[inline]
+    fn might_complex_op<F>(&self, other: PyObjectRef, op: F, vm: &VirtualMachine) -> PyResult
```

Comment:
I think this name can be simply `complex_op`.
```suggestion
        fn complex_op<F>(&self, other: PyObjectRef, op: F, vm: &VirtualMachine) -> PyResult
```